server/models/Notification.js

```diff
@@ -132,6 +132,31 @@ export default function(Sequelize, DataTypes) {
       },
     }).then(us => us.map(us => us.UserId));
   };
+  /**
+   * Check if notification with `NotificationType` and `UserId` is active.
+   * @param {*} CotificationType
+   * @param {*} UserId
+   * @param {*} [Collective]
+   */
+  Notification.isActive = (NotificationType, UserId, Collective) => {
```

Comment:
It's strange to have `UserId` (an id) in one case and `Collective` (an object) in the other. Can you reconsider that?

Reply:
Yeah, make both object.